vnext/ReactUWP/Views/cppwinrt/BorderEffect.idl

```diff
@@ -5,21 +5,32 @@
 //
 // If you modify this file, be sure to run buildcppwinrt.cmd in this folder
 // to generate new headers
+#include"../../../include/NamespaceRedirect.h"
 
+#ifdef NO_WINUI3_SUPPORT
 namespace Microsoft.UI.Composition.Effects
+#else
+namespace Microsoft.ReactNative
+#endif
 {
+#ifdef NO_WINUI3_SUPPORT
     [version(1)]
     typedef enum CanvasEdgeBehavior
     {
         Clamp = (int)0,
         Wrap = (int)1,
         Mirror = (int)2
     } CanvasEdgeBehavior;
+#endif
 
     runtimeclass BorderEffect;
 
     [version(1)]
+#ifdef NO_WINUI3_SUPPORT
     [uuid(31602441-15db-5b4a-98dd-ba4247548b40), exclusiveto(BorderEffect)]
+#else
+    [uuid(d21ed48b-9d10-40d1-a6f6-e6bfca14a050), exclusiveto(BorderEffect)]
```

Comment:
>uuid [](start = 5, length = 4)

Does this GUID have a special meaning? I wonder if the whole file can be rewritten in MIDL 3.0.

Reply:
the WinUI 2.x version matches a certain internal clsid in Win2D.
The WinUI3 one is randomly autogenerated.

(these are WinRT types so it should be possible to upgrade this, but this code matches some internal WinUI code so I'm not touching it for the time being)